subgraphs/lido/schema.graphql

```diff
@@ -439,13 +439,13 @@ type PoolHourlySnapshot @entity {
 
 # An account is a unique Ethereum address
 # Helps to accumulate total unique users
-type Account @entity {
+type Account @entity @regularPolling {
   " Address of the account "
   id: ID!
 }
 
 # Helper entity for calculating daily/hourly active users
-type ActiveAccount @entity {
+type ActiveAccount @entity @regularPolling {
```

Comment:
Do we want this? Essentially shows unique users on a day/hour

Reply:
Yeah i think this will be useful to have, but it does sound like there'll be a ton of data we're polling regularly in that case.

Reply:
The count of `ActiveAccount` is probably 60-80% of all transactions ? so quite a bit